packages/shared/src/lib/core/market/actions/getAndUpdateShimmerEvmTokensImages.ts

```diff
@@ -0,0 +1,23 @@
+import { CoinGeckoApi } from '../apis'
+import { CoinGeckoCoinImage } from '../interfaces'
+import { shimmerEvmTokensImages } from '../stores'
+
+export async function getAndUpdateShimmerEvmTokensImages(): Promise<void> {
+    try {
+        const coinGeckoShimmerEvmTokens = await CoinGeckoApi.getFilteredCoinsList()
+        const tokenImagePromises = coinGeckoShimmerEvmTokens.map(async (token) => {
+            const tokenDetails = await CoinGeckoApi.getCoinDetails(token.id)
+            return { [token.platforms.shimmer_evm]: tokenDetails.image ?? {} }
```

Comment:
why dont we store the whole metadata and not just image, might be useful soon

Reply:
Whether it's useful or not, why add data that we don't know we'll use? It's better to implement on demand than have more technical debt

Reply:
its harder to update the persisted store without migration?